airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/StatusMainInfo.tsx

```diff
@@ -15,19 +15,10 @@ import { useSourceDefinition } from "services/connector/SourceDefinitionService"
 import EnabledControl from "./EnabledControl";
 import styles from "./StatusMainInfo.module.scss";
 
-interface StatusMainInfoProps {
-  connection: WebBackendConnectionRead;
-  source: SourceRead;
-  destination: DestinationRead;
-  onStatusUpdating?: (updating: boolean) => void;
-}
-
-export const StatusMainInfo: React.FC<StatusMainInfoProps> = ({
-  onStatusUpdating,
-  connection,
-  source,
-  destination,
-}) => {
+export const StatusMainInfo: React.FC = () => {
+  const {
+    connection: { source, destination, status },
```

Comment:
Similar question here - could this just use `useGetConnection` instead of `useConnectionEditService`?

Reply:
See previous reply.